### PR TITLE
Chara Maker, Clothes and Items Translations

### PR DESCRIPTION
**Pull request by SpockBauru**
Clothes, and Items Translation from list/characustom (Almost 200 files!)
Created a Character_Maker.txt following the zz_AI template. It also have a list of items files according its section.
Replaced Maker.txt with Character_Maker_regex.txt
Many clothes and items were translated according their ingame appearance, and not the direct translation. Also tried to match sets names such as succubus set, sister set, etc.

### CODE REVIEW

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/00/ao_ear_00/translation.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-スター=Star
-スターリング=Sterling
-クリスタル=Crystal
-クリスタルピラー=Crystal Pillar
-三連イヤリング=Triple Earrings
-フラワー=Flower
-パール=Pearl
+スター=Star earring
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/list/characustom/00/ao_ear_00/translation.txt:1]: I think generally things should be `Title Case` so `Star Earring` (easy enough to me to fix up after it's merged if you'd rather not).

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/00/st_eye_hl_00/translation.txt [hunk 2]
```diff
@@ -42,4 +42,4 @@
 タイプ42=Type 42
 タイプ43=Type 43
 タイプ44=Type 44
-//光源反射=
+光源反射=Reflection of the Light Source
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/list/characustom/00/st_eye_hl_00/translation.txt:45]: This seems a bit wordy, would `Light Reflection` work?

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/00/st_paint_00/translation.txt [hunk 3]
```diff
@@ -2,38 +2,38 @@
...
+禁止=Forbidden
 陰陽=Yin-Yang
 バラ=Rose
-キスマーク=Kiss Mark
-瞳=Pupil
-//瞳２=
-汚れ=Dirt
-四角=Rectangular
+キスマーク=Kiss
+瞳=Eye 1
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/list/characustom/00/st_paint_00/translation.txt:16]: `Eye` to be consistent with original?

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/07/ao_arm_07/translation.txt [hunk 4]
```diff
@@ -2,9 +2,9 @@
 メカ装甲５=Mecha Armor 5
 メカ装甲６=Mecha Armor 6
 メカ装甲６×３=Mecha Armor 6×3
-メカブースターⅡ=Mecha Booster Ii
-メカシールドⅡ=Mecha Shield Ii
-メカシールドⅢ=Mecha Shield Ⅲ
+メカブースターⅡ=Mecha Booster 2
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/list/characustom/07/ao_arm_07/translation.txt:5]: Maybe just keep the special character? `Mecha Booster Ⅱ`

File: Translation/en/Text/Main/Character_Maker.txt [hunk 5]
```diff
@@ -0,0 +1,1126 @@
+//***********************************************************************
+//***********************************************************************
+// Note: Some text is intentionally cut short or have thier context
+//       removed to fit the UI.
+//           Eyebrow Height -> Height // this is inside the Eyebrow menu
+//           Left Shoulder -> L Shoulder
+//***********************************************************************
+//***********************************************************************
+
+// Scene: CharaCustom (4)
```
Comment (GeBo1) [inline Translation/en/Text/Main/Character_Maker.txt:10]: Update the comment to say `CharaCustom (3)`

File: Translation/en/Text/Main/Character_Maker.txt [hunk 6]
```diff
@@ -0,0 +1,1126 @@
...
+
+
+//------------------------------------------------------------
+// Patterns
+//------------------------------------------------------------
+// RedirectedResources\assets\abdata\list\characustom\00\st_pattern_00\
+// RedirectedResources\assets\abdata\list\characustom\30\st_pattern_30\
+// RedirectedResources\assets\abdata\list\characustom\34\st_pattern_34\
+
+八つ藤=yatsu fuji
```
Comment (GeBo1) [inline Translation/en/Text/Main/Character_Maker.txt:262]: Why is this here instead of up in `RedirectedResources\assets\abdata\list\characustom\34\st_pattern_34\`?

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/34/st_pattern_34/translation.txt [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
-//毬=
-//菊=
+牡丹=Peony
+雪輪=Round Snow
+毬=Bouquet
+菊=Chrysanthemum
 桜=Cherry Blossoms
-//霞桜=
+霞桜=Falling Cherry Blossoms
 //八つ藤=
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/list/characustom/34/st_pattern_34/translation.txt:7]: Why is this one translated in `Text/Main/Character_Maker.txt`?

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/00/ao_ear_00/translation.txt (hunk 1 above)
Comment (SpockBauru) [reply Translation/en/RedirectedResources/assets/abdata/list/characustom/00/ao_ear_00/translation.txt:1]: Totally forgot about `Title Case`, there should be lots of errors like this, sorry...

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/00/st_eye_hl_00/translation.txt (hunk 2 above)
Comment (SpockBauru) [reply Translation/en/RedirectedResources/assets/abdata/list/characustom/00/st_eye_hl_00/translation.txt:45]: Yep, `Light Reflection` is indeed better!

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/07/ao_arm_07/translation.txt (hunk 4 above)
Comment (SpockBauru) [reply Translation/en/RedirectedResources/assets/abdata/list/characustom/07/ao_arm_07/translation.txt:5]: This one was intentional, since other mecha set don't have Roman numbers

File: Translation/en/Text/Main/Character_Maker.txt (hunk 5 above)
Comment (SpockBauru) [reply Translation/en/Text/Main/Character_Maker.txt:10]: Ok, I just don't know how to do it here... This github thing is totally new for me

File: Translation/en/Text/Main/Character_Maker.txt (hunk 6 above)
Comment (SpockBauru) [reply Translation/en/Text/Main/Character_Maker.txt:262]: Its because I don't have idea how to translate this pattern, so I leave there for further investigations

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/34/st_pattern_34/translation.txt (hunk 7 above)
Comment (SpockBauru) [reply Translation/en/RedirectedResources/assets/abdata/list/characustom/34/st_pattern_34/translation.txt:7]: Its because I don't have idea how to translate this pattern, so I leave there for further investigations

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/00/st_paint_00/translation.txt (hunk 3 above)
Comment (SpockBauru) [reply Translation/en/RedirectedResources/assets/abdata/list/characustom/00/st_paint_00/translation.txt:16]: Ok, just did this because I like numbering things :D

File: Translation/en/Text/Main/Character_Maker.txt (hunk 5 above)
Comment (SpockBauru) [reply Translation/en/Text/Main/Character_Maker.txt]: Tried to make like Anon11 said, now should be corrected

**Comment by GeBo1** [comment]
Reopening

File: Translation/en/Text/Main/Character_Maker.txt (hunk 6 above)
Comment (GeBo1) [reply Translation/en/Text/Main/Character_Maker.txt:262]: Ok, then we should leave it untranslated then, but add a note in the pattern_34 file about it.

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/34/st_pattern_34/translation.txt (hunk 7 above)
Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/list/characustom/34/st_pattern_34/translation.txt:7]: add a comment here then, like:
```
// I have no idea how to translate this one
//八つ藤=
```

File: Translation/en/RedirectedResources/assets/abdata/list/characustom/07/ao_arm_07/translation.txt (hunk 4 above)
Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/list/characustom/07/ao_arm_07/translation.txt:5]: I'd prefer they match the original, but you did the work so I'm fine with it.
